app/src/main/java/com/projectit3048c/dto/FoodNutrients.kt

```diff
@@ -1,3 +1,11 @@
 package com.projectit3048c.dto
 
-data class FoodNutrients(var nutrientId: Int = 0, var nutrientName: String = "nothing", var nutrientNumber: Double = 0.0, var value: Double = 0.0)
+data class FoodNutrients(var nutrientId: Int = 0,
+                         var nutrientName: String = "nothing",
+                         var nutrientNumber: Double = 0.0,
+                         var value: Double = 0.0
+) {
```

Comment:
Long class headers should have parameters in a separate line.
https://kotlinlang.org/docs/coding-conventions.html#class-headers